universe/auto_syncer.go

```diff
@@ -228,27 +235,16 @@ func (f *FederationEnvoy) syncServerState(ctx context.Context,
 func (f *FederationEnvoy) pushProofToFederation(uniID Identifier, key LeafKey,
 	leaf *Leaf) {
 
-	ctx, cancel := f.WithCtxQuit()
-	defer cancel()
-
-	fedServers, err := f.cfg.FederationDB.UniverseServers(
-		ctx,
-	)
-	if err != nil {
-		err := fmt.Errorf("unable to fetch set of universe "+
-			"servers: %v", err)
-		f.reportErr(err)
```

Comment:
I have a commit that removes this report, otherwise it causes the daemon to shutdown.